ifrs/core/file_registry.py

```diff
@@ -57,18 +58,32 @@ async def _is_file_registered(self, *, file: models.FileMetadata) -> bool:
         except ResourceNotFoundError:
             return False
 
-        if file == registered_file:
+        # object ID is a UUID generated upon registration, so cannot compare those
+        file_without_object_id = file.dict(exclude={"object_id"})
```

Comment:
If `file` is of type `FileMetadataBase` it should not have an object_id, right?
So this should not need the exclude

Reply:
100% correct